src/strongbus_spec.ts

```diff
@@ -1506,158 +1506,98 @@ describe('Strongbus.Bus', () => {
         expect(onResolve).toHaveBeenCalledWith(true);
 
       });
+    });
 
-      describe('and params.eager=false', () => {
-        it('does not resolve the promise until it receives an event triggering evaluation', async () => {
-          const hasFoo: boolean = true;
-          const evaluator = (resolve: Scanner.Resolver<boolean>, reject: Scanner.Rejecter) => {
-            if(hasFoo) {
-              resolve(true);
-            }
-          };
-          const p = bus.scan({
-            evaluator,
-            trigger: 'foo',
-            eager: false
-          });
+    describe('and params.eager=false', () => {
+      it('does not resolve the promise until it receives an event triggering evaluation', async () => {
+        const hasFoo: boolean = true;
+        const evaluator = (resolve: Scanner.Resolver<boolean>, reject: Scanner.Rejecter) => {
+          if(hasFoo) {
+            resolve(true);
+          }
+        };
+        const p = bus.scan({
+          evaluator,
+          trigger: 'foo',
+          eager: false
+        });
 
-          p.then(onResolve);
-          await sleep(1);
+        p.then(onResolve);
+        await sleep(1);
 
-          expect(onResolve).not.toHaveBeenCalled();
+        expect(onResolve).not.toHaveBeenCalled();
 
-          bus.emit('foo', 'FOO!');
-          await sleep(1);
+        bus.emit('foo', 'FOO!');
+        await sleep(1);
 
-          expect(onResolve).toHaveBeenCalledWith(true);
-        });
+        expect(onResolve).toHaveBeenCalledWith(true);
       });
+    });
 
-      describe('Scanner pooling', () => {
-        describe('given multiple scan invocations in which params.evaluator and params.eager are the same', () => {
-          describe('given params.trigger is the same single event', () => {
-            it('returns the same promise object', () => {
-              const evaluator = (resolve: Scanner.Resolver<boolean>, reject: Scanner.Rejecter) => {
-                // doing nothing in the evaluator
-                return;
-              };
-
-              const p1 = bus.scan({
-                evaluator,
-                trigger: 'foo'
-              });
-              const p2 = bus.scan({
-                evaluator,
-                trigger: 'foo'
-              });
-
-              expect((p1 as any)[INTERNAL_PROMISE] === (p2 as any)[INTERNAL_PROMISE]).toBeTrue();
-            });
-          });
+    describe('Scanner pooling', () => {
```

Comment:
Scanner pooling specs were mis-indented. See other comment to locate new spec for `params.timeout`